### ISSUE TITLE
Misspelling?

### ISSUE THREAD

**Issue by Osalotioman** [GDGoC-University-of-Benin/the-intern-place#7]
The repo name is spelt:
the-intern-placee
Is this a misspelling for:
the-intern-place
If yes, why not change the name of the repo?

**Comment by d-a-ve** [comment]
No, it is not a misspelling. another repo with the name `the-intern-place` was created under this account and now another repo with the same name cannot be created even though it has been deleted.

**Comment by d-a-ve** [comment]
I am closing this issue now since I have answered the question.
